docs/guides/core-concepts/cypress-app.mdx

```diff
@@ -172,6 +172,36 @@ The title of each run is taken from the
 for that change, and clicking on it will take you to the corresponding run page
 in Cypress Cloud.
 
+### Debug
+
+<DocsImage
+  src="/img/guides/core-concepts/cypress-app/debug-page.png"
+  alt="Debug"
+></DocsImage>
+
+The Debug page allows you to easily debug failed CI test runs from within the
+Cypress app. From this page, you can review, rerun, and debug failed CI test
+runs that are recorded to Cypress Cloud (see [note above](#Overview)) – all
+without leaving Cypress.
+
+This feature eliminates the tedious process of switching between Cypress, your
+local editor, and CI output, in order to fix failed tests. It does this by
+enabling you to use the [Test Runner](#The-Test-Runner) to run only the tests
+that failed in your last recorded test run, as well as review screenshots,
+videos, and logs from your tests.
+
+The Debug page shows the latest completed [test run](/guides/cloud/runs) that
+matches your current locally checked out git commit. If no run is found for your
```

Comment:
I know we've gone around a bit on the terminology here. I'll throw out an alternative but no strong feelings either way:

```suggestion
matches the HEAD commit of your local project. If no run is found for your
```